airbyte-integrations/connectors/destination-gcs/build.gradle

```diff
@@ -12,6 +12,7 @@ application {
 dependencies {
     implementation project(':airbyte-config:models')
     implementation project(':airbyte-protocol:models')
+    implementation project(':airbyte-workers')
```

Comment:
It looks like the worker is not needed?

Reply:
Hi @tuliren.
I've added a new test for insufficient roles verification that calls runCheck method. If fails like this if airbyte-workers haven't been added. 
![Selection_365](https://user-images.githubusercontent.com/4688354/148651899-b68b4831-649f-4c4f-ad37-ae5b4239f054.png)


But you're right, that this is not required for the whole project, moved it to integrationTestJavaImplementation dependency.
Thanks